packages/playground/__tests__/context/messages/reducer.test.tsx

```diff
@@ -0,0 +1,89 @@
+import { reducer, defaultState, Action } from '~/context/messages/reducer';
+
+describe('Given the reducer is used to manage state', () => {
+  describe('When "LOG_SYSTEM" action is dispatched', () => {
+    test('When message of type "IN_PROGRESS" is dispatched', () => {
+      const status = 'IN_PROGRESS';
+      const content = 'System message content';
+      const payload: Action = {
```

Comment:
this is an `action`, not a `payload`, a `payload` is a part of an `action`. Therefore, we should call it `action`